include/ck/utility/f8_utils.hpp

```diff
@@ -6,8 +6,6 @@
 #include "ck/utility/data_type.hpp"
 
 // these conversions are disabled if native conversions available
-#if !defined(__gfx940__) && !defined(__gfx941__) && !defined(__gfx942__)
```

Comment:
We might want to keep this condition because on MI300 we use native conversion intrinsics and need these casting functions on all other architectures.

Reply:
We have a condition in vector_type.